src/DevSoc.Eventathon/Models/EventRestResponse.cs

```diff
@@ -0,0 +1,8 @@
+﻿using DevSoc.Eventathon.Models;
+namespace DevSoc.Eventathon.Calendars.Models;
+
+public class EventRestResponse
+{
+    public EventResponse @event { get; set; }
```

Comment:
Why the wrapper? It's redundant

Reply:
Good point. I added that whilst half-asleep overnight so I really don't know what I was thinking.